README.md

```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
 
 ## :tomato: Compatibility
 Usually the plugin is not affected by small updates of Telegram.  
-But some updates may cause the plugin to error or crash, please [report the issue](https://github.com/SpriteOvO/Telegram-Anti-Revoke/issues) and wait for the plugin to be updated.
+But some updates may cause the plugin to error or crash, please [report the issue](/issues) and wait for the plugin to be updated.
```

Comment:
Relative links in `README.md` are relative to the repository directory, this change will cause it to jump to `https://github.com/SpriteOvO/Telegram-Anti-Revoke/blob/dev/issues`, it should be reverted.

Reply:
Got it.